setup/helpers/perf_helper.rb

```diff
@@ -307,80 +335,72 @@ def puppet_module_dependencies
     # Hacky, but new versions of puppet require different stuff in Puppetfile,
     # for older versions, remove those lines.
     step "Use external puppet modules for puppet >= 6.0" do
-      puppet_version = on(master, 'puppet --version').stdout
+      staging_prod = "/etc/puppetlabs/code-staging/environments/production"
+      puppet_version = on(master, "puppet --version").stdout
       if master.version_is_less(puppet_version, "6.0")
-        on master, "sed -i '/puppetlabs\\/.*_core/d' /etc/puppetlabs/code-staging/environments/production/Puppetfile"
-        on master, "rm -rf /etc/puppetlabs/code-staging/environments/production/modules/*_core"
+        on master, "sed -i '/puppetlabs\\/.*_core/d' #{staging_prod}/Puppetfile"
+        on master, "rm -rf #{staging_prod}/modules/*_core"
       end
     end
   end
 
   def enable_file_sync
     api = Scooter::HttpDispatchers::ConsoleDispatcher.new(dashboard)
-    pe_master_group = api.get_node_group_by_name('PE Master')
-    pe_master_group['classes']['puppet_enterprise::profile::master']['file_sync_enabled'] = true
-    api.replace_node_group(pe_master_group['id'], pe_master_group)
-    on(master, 'puppet agent -t', :acceptable_exit_codes => [0,2])
+    pe_master_group = api.get_node_group_by_name("PE Master")
+    pe_master_group["classes"]["puppet_enterprise::profile::master"][
+      "file_sync_enabled"] = true
+    api.replace_node_group(pe_master_group["id"], pe_master_group)
+    on(master, "puppet agent -t", acceptable_exit_codes: [0, 2])
   end
 
   def sync_code_dir
```

Comment:
I like this update! However, it removes the comments that explain what is going on here. Can you please add a comment header that explains it?

Reply:
I have added the comment back as the method doc. Thanks.